Doc/library/socketserver.rst

```diff
@@ -294,14 +288,35 @@ Server Objects
 
    .. XXX should class variables be covered before instance variables, or vice versa?
 
+   .. attribute:: address_family
+
+      The family of protocols to which the server's socket belongs.
+      Common examples are :const:`socket.AF_INET` and :const:`socket.AF_UNIX`.
+
+
    .. attribute:: allow_reuse_address
 
       Whether the server will allow the reuse of an address.  This defaults to
       :const:`False`, and can be set in subclasses to change the policy.
 
 
+   .. attribute:: allow_reuse_port
+
+      Whether the server will allow the reuse of a port. This defaults to
+      :const:`False`, and can be set in subclasses to change the policy.
+
+   .. attribute:: max_packet_size
+
+      Only for datagram sockets, e.g. UDP.
+      The maximum amount of data (in bytes) to receive at once. If a packet is
+      too large for the buffer, then the excess bytes beyond :attr:`max_packet_size`
+      are discarded. The default value is usually 8192, but this can be
```

Comment:
Might be worth code-blocking this to make it stand out. Happy to yield to you, though.
```suggestion
      are discarded. The default value is usually ``8192``, but this can be
```

Reply:
sure, lets do that for the 5 as well for consistency

Reply:
done, looks better